build.gradle

```diff
@@ -41,6 +41,9 @@ buildscript {
 
         // Check latest version of dependencies, task: dependencyUpdates
         classpath gradlePlugins.versions
+
+        // Analyse builds to profile
+        classpath gradlePlugins.buildScan
```

Comment:
Looks like this plugin eats too much memory and CI fails with error code `137` which means OutOfMemoryError.

Can you please try to tune memory usage to pass the CI?